publications-2018.bib

```diff
@@ -1231,17 +1271,6 @@ @Article{lee2018optimal
 publisher={Springer International Publishing}
 }
 
-@Article{lee2018phase,
-title={Phase-field modeling of two phase fluid filled fractures in a poroelastic medium},
```

Comment:
Did this entry exist twice, or what was the reason for removing it?

Reply:
Yes. It was listed as both Lee2018 and lee2018phase